R/utils.R

```diff
@@ -0,0 +1,20 @@
+#' Convert a list of lists to dataframe
+#'
+#' Convert a list of lists to dataframe; requires that every list have same named elements.
+#'
+#' @param my_list A list of lists to dataframe
+#' @param as_character A logical value, indicating whether the values are read as character
+#' @param on_empty Value to return if my_list is NULL, NA or is length == 0, default = NA
+#'
+#' @export
+#' @examples util_list_to_df2(util_df_to_list(dplyr::starwars))
```

Comment:
This example is causing tests to fail because `austraits` does not have access to `util_df_to_list()`. Is there another example we can use? Alternatively, removing the @example also works.